python/ray/data/tests/test_mars.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+import ray
+import mars
+import mars.tensor as mt
+import mars.dataframe as md
+
+
+@pytest.fixture(scope="module")
+def ray_start_regular(request):  # pragma: no cover
+    param = getattr(request, "param", {})
+    if not param.get("enable", True):
+        yield
+    else:
+        num_cpus = param.get("num_cpus", 64)
+        try:
+            yield ray.init(num_cpus=num_cpus)
+        finally:
+            ray.shutdown()
```

Comment:
Mars use ray actors to execute computing. If not specified enough cpus, the ray dataset may never gets enough cpus for execution. `64` is too much, I set it to 16 since some ci machein amy have only 1 cpu..